docs/guides/admin/docs/workflowoperationhandlers/composite-woh.md

```diff
@@ -22,6 +26,7 @@ Tags and flavors can be used in combination.
 
 |configuration keys|value type (EBNF)|example|description|default value|
 |------------------|-----------------|-------|-----------|-------------|
+|source-audio-name|String , { "," , String }    |upper|The "name" of track to use as a source audio.|EMPTY|
```

Comment:
Good catch in the value type!.
I thought a lot about the name. If I use a flavor or tag, then I have to worry about extra tracks in the mediapackage that matches the tag/flavor but is not "lower" or "upper".  I can't use tag or key or flavor, so I use name, because "upper" and "lower" are used throughout and I see them as named inputs. If I just use source-audio, then it is even more ambiguous.  I will add "both" though.

Reply:
Actually I guess omission is a "don't care", since compose will try and use both if they exist or any one if only one has audio.

Reply:
Agree on omission being a "don't care", but still weird to only be able to get the default by omitting the configuration key. Finally, "don't care" is a valid option. Why should I be unable to explicitly set this? But no strong opinion ;-)

Also no strong opinion on the name of the configuration key.